modules/index.d.ts

```diff
@@ -206,6 +206,11 @@ export interface BuilderProps {
   dispatch: Dispatch,
 }
 
+export interface ItemBuilderProps extends BuilderProps {
+  type: "rule" | "group" | "rule_group" | "switch_group" | "case_group";
+  itemComponent: ReactElement;
```

Comment:
Not ReactElement, but Factory

Reply:
> Not ReactElement, but Factory

Thanks for your comments
update PR based on your comments, hope this time all fine.